### ISSUE TITLE
Screen RGB : Size

### ISSUE THREAD

**Issue by ArnaudMartinENSI** [logisim-evolution/logisim-evolution#1410]
Hi,
Is-it possible to add size for the screen RGB.
I would like try a vga controler, and i need a screen with 800*600 pixels.

And is it possible to add others color models (like 444).

Have a good day.

**Comment by BFH-ktt1** [comment]
Any PR are welcomed on this, but it is not likely to be implemented by us in the near future.

**Comment by MarcinOrlowski** [comment]
>  the screen RGB.

@ArnaudMartinENSI  Do you mean "RGB Video" component?

**Comment by ArnaudMartinCaen** [comment]
@MarcinOrlowski Yes in french the name is "Ecran RGB". I would like try a vga controller (800*600) with this screen before test on hardware.

**Comment by MarcinOrlowski** [comment]
@ArnaudMartinENSI Unfortunatelly both are named the same :( Is the one you talk about in "Input/Output" section (that's RGB Video) or rather in "System on a chip" (SOC) (that's VGA Screen)?

**Comment by ArnaudMartinCaen** [comment]
@MarcinOrlowski Yes it's the RGB Video in the 'Input/Output" section

**Comment by MarcinOrlowski** [comment]
As this element is FPGA supported, let me call @BFH-ktt1 and @maehne to join and discussion if we can get beyond 256x256 there also in context of FPGA or there's unbeatable obstacle in that field.

**Comment by maehne** [comment]
At least in my local build of Logisim-evolution both components, "RGB Video" from library "Input/Output" and "VGA screen" from the library "System On a Chip", are not supported for synthesis. Supporting this well for different resolutions and colour depths would be challenging to implement to achieve the correct pixel clock frequency and signal timing.

The SOC VGA screen supports higher resolutions, but lacks documentation like the RGB Video component from the Input/Output library. So, it should be doable to at least support it for simulation for the RGB Video component, too. PRs in this direction are welcome!

**Comment by hewzhew** [comment]
I checked this against current `main`. The resolution part of this request appears to have been completed by merged PR #2386:

- RGB Video now provides several preset resolutions plus a `Custom` resolution;
- custom width and height accept values from 2 through 4096; and
- the X/Y input widths are derived from the selected dimensions, so the requested 800x600 simulation use case uses 10-bit coordinates and is supported.

The other request in the issue, an RGB444 color mode, is independent and is not among the current color options.

Would it make sense to treat the resolution request as completed and, if RGB444 is still wanted, track that as a focused follow-up issue? This would avoid duplicating the already-merged resolution work while keeping the remaining color-format request visible.

**Comment by maehne** [comment]
I agree, let's close this issue as resolved. I'll create a follow up issue.
